mmengine/hooks/logger_hook.py

```diff
@@ -305,6 +329,6 @@ def after_run(self, runner) -> None:
                 f'{out_filepath}.')
 
             if not self.keep_local:
-                os.remove(local_filepath)
                 runner.logger.info(f'{local_filepath} was removed due to the '
                                    '`self.keep_local=False`')
+                os.remove(local_filepath)
```

Comment:
Is it necessary to move this line after `logger.info` because `FileHandler` has been removed?

Reply:
Otherwise `runner.logger` cannot exit normally in Windows.